ts/packages/cli/src/commands/export.ts

```diff
@@ -213,26 +267,67 @@ export default class Export extends BaseCommand<typeof Export> {
     return caseFunction ? _.mapKeys(result, (value, key) => caseFunction(key)) : result;
   }
 
+  filterFromFlags(): (({ context, result }: EvalCommandReturn['string']) => boolean) | undefined {
+    const filterFlags = _.pickBy(this.flags, (value, key) => Object.values(FilterFlag).includes(key as FilterFlag));
+    if (!_.isEmpty(filterFlags)) {
+      const pickFlags = _.pick(filterFlags, [FilterFlag.PickLabel, FilterFlag.PickKey]);
+      const omitFlags = _.pick(filterFlags, [FilterFlag.OmitLabel, FilterFlag.OmitKey]);
+      return ({ context, result }: EvalCommandReturn['string']): boolean => {
```

Comment:
NIT: the implementation is solid but the usage of reducers makes this code verbose and hard to read. How do you feel about this suggestion:
```node.js
    const pickedKeys = (_.pick(filterFlags, [FilterFlag.PickKey])['pick-key'] as string[]) ?? [];
    const pickedLabels = (_.pick(filterFlags, [FilterFlag.PickLabel])['pick-label'] as string[]) ?? [];
    const omittedKeys = (_.pick(filterFlags, [FilterFlag.OmitKey])['omit-key'] as string[]) ?? [];
    const omittedLabels = (_.pick(filterFlags, [FilterFlag.OmitLabel])['omit-label'] as string[]) ?? [];

    return ({ context, result }: EvalCommandReturn['string']): boolean => {
      const hiddenFilter = this.flags[FilterFlag.PickHidden] ? true : !context.cfgu.hidden;
      const emptyFilter = this.flags[FilterFlag.OmitEmpty] ? result.origin !== EvaluatedConfigOrigin.EmptyValue : true;

      const isKeyPicked = pickedKeys.includes(context.key);
      const isLabelPicked = _.intersection(pickedLabels, context.cfgu.labels ?? []).length > 0;
      const pickFilter = _.isEmpty([...pickedKeys, ...pickedLabels]) || isKeyPicked || isLabelPicked;

      const isKeyOmitted = omittedKeys.includes(context.key);
      const isLabelOmitted = _.intersection(omittedLabels, context.cfgu.labels ?? []).length > 0;
      const omitFilter = !isKeyOmitted && !isLabelOmitted;

      return hiddenFilter && emptyFilter && pickFilter && omitFilter;
    };
```

Feel free to take whichever part of it you like if any.

Reply:
I prefer to use vanilla instead of lodash every day.
The only reason I use it here is because eslint doesn't allow to reduce but misses _.reduce.
For me, it's harder to read your code than mine... :/

But, I'm not in love with mine. If it matters to you i'll use yours.